src/browser/provider/built-in/dedicated/chrome/index.js

```diff
@@ -21,9 +21,13 @@ export default {
         return cdp;
     },
 
+    async _createRunTimeInfo (hostName, configString, allowMultipleWindows) {
+        return await ChromeRunTimeInfo.create(hostName, configString, allowMultipleWindows);
```

Comment:
`await` is redundant here.

Reply:
IMHO, we should enable the [no-return-await](https://eslint.org/docs/rules/no-return-await) ESLint rule, but there are many places that use the problematic pattern. Let's add it to our refactoring backlog https://github.com/DevExpress/testcafe/issues/4635